cdap-app-fabric/src/main/java/io/cdap/cdap/internal/tethering/TetheringServerHandler.java

```diff
@@ -93,17 +95,17 @@ public void init(HandlerContext context) {
   }
 
   /**
-   * Sends control commands to the client. Also receives program status updates from the client.
+   * Sends control commands to the client.
```

Comment:
This is used to receive program status updates from the client.

Unless there's some other place where it's used to send commands?

Reply:
updated javadoc